lib/ansible/modules/system/pid.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: pid
+version_added: 2.8
+description: "Gives the pid of the given process name"
+short_description: "Gives the pid of the given process name"
+author:
+  - Saranya Sridharan
+requirements:
+  - only standard library needed
+options:
+  name:
+    description: the name of the process you want to get pid for
+    required: true
+    default: null
+'''
+EXAMPLES = '''
+# Pass the process name
+- name: Getting pid of the process
+  pid:
+      name: python
+'''
+RETURN = '''
+pid:
+  description: It return process id of the given process
+  returned: list of process id
+  type: dict
+'''
+
+import subprocess
+from ansible.module_utils.basic import AnsibleModule
+
+
+def get_pid(name):
+    processid = subprocess.check_output("pidof %s" % (name), shell=True)
+    return processid.split("\n")
```

Comment:
On my machine (Linux), the output of `pidof` is separated by spaces, not by newlines.

Also, when `name` contains spaces, this doesn't work. It's better to pass a list of strings like this: `["pidof", name]`, instead of a precomposed string.